### ISSUE TITLE
Loading large BNDB in 3.1.3578-dev hangs the UI after the loading dialog finishes

### ISSUE THREAD

**Issue by alexrp** [Vector35/binaryninja-api#3332]
So far, this is only happening on the main BNDB I'm working on which is ~346 MB.

I attached WinDbg during the hang and got this stack trace:

```
[0x0]   Qt6Core!QCoreApplicationPrivate::removePostedTimerEvent + 0x8c   
[0x1]   Qt6Core!QEventDispatcherWin32Private::unregisterTimer + 0x1e   
[0x2]   Qt6Core!QEventDispatcherWin32::unregisterTimer + 0xd0   
[0x3]   Qt6Core!QBasicTimer::stop + 0x2c   
[0x4]   Qt6Core!QBasicTimer::start + 0xf7   
[0x5]   Qt6Widgets!QAbstractItemView::qt_static_metacall + 0x22f   
[0x6]   Qt6Core!QObject::qt_static_metacall + 0x17cc   
[0x7]   Qt6Core!QMetaObject::activate + 0x84   
[0x8]   Qt6Core!QAbstractItemModel::endInsertRows + 0x107   
[0x9]   binaryninjaui!StringsListModel::updateStrings + 0x2dd   
[0xa]   Qt6Core!QObject::qt_static_metacall + 0x1761   
[0xb]   Qt6Core!QMetaObject::activate + 0x84   
[0xc]   Qt6Core!QTimer::timerEvent + 0x77   
[0xd]   Qt6Core!QObject::event + 0x97   
[0xe]   Qt6Widgets!QApplicationPrivate::notify_helper + 0x10d   
[0xf]   Qt6Widgets!QApplication::notify + 0x187f   
[0x10]   Qt6Core!QCoreApplication::notifyInternal2 + 0xc4   
[0x11]   Qt6Core!QEventDispatcherWin32Private::sendTimerEvent + 0x107   
[0x12]   Qt6Core!QEventDispatcherWin32::processEvents + 0xc1a   
[0x13]   USER32!UserCallWinProcCheckWow + 0x37e   
[0x14]   USER32!DispatchMessageWorker + 0x1f1   
[0x15]   Qt6Core!QEventDispatcherWin32::processEvents + 0x509   
[0x16]   Qt6Gui!QWindowsGuiEventDispatcher::processEvents + 0x19   
[0x17]   Qt6Core!QEventLoop::exec + 0x19e   
[0x18]   Qt6Core!QCoreApplication::exec + 0x15c   
[0x19]   binaryninja + 0x46683   
[0x1a]   binaryninja + 0x24caa4   
[0x1b]   KERNEL32!BaseThreadInitThunk + 0x1d   
[0x1c]   ntdll!RtlUserThreadStart + 0x28   
```

I'll try to narrow down the dev build where this started.

**Comment by alexrp** [comment]
3.1.3576-dev works fine, 3.1.3577-dev has the bug.

**Comment by alexrp** [comment]
Here's what I see after the loading dialog disappears and the UI hangs:

![image](https://user-images.githubusercontent.com/44076/181385322-5cdaef6d-c661-451d-9a7c-af729b25e64b.png)

**Comment by bpotchik** [comment]
Can you share your bndb by chance? I already made one with the tera.exe provided earlier but was unable to reproduce the issue so far...

**Comment by bpotchik** [comment]
Also, do you have any additional plugins, custom binaryviews, or workflows that are installed/enabled?

**Comment by alexrp** [comment]
> Can you share your bndb by chance? I already made one with the tera.exe provided earlier but was unable to reproduce the issue so far...

Here it is: https://drive.google.com/open?id=1IXi4JeqD2lst4vihBPiZw-ybVdUuYl3m

**Comment by alexrp** [comment]
I do have some plugins, but the issue occurs even with all of them removed.

**Comment by bpotchik** [comment]
Interesting. Okay this problem did not reproduce on a mac, however you're bndb (not mine) did repro on a windows box.

**Comment by bpotchik** [comment]
Thanks for the report!

Fixed in `3.1.3580-dev`.
